collector/Makefile.Common

```diff
@@ -0,0 +1,55 @@
+# ALL_PKGS is the list of all packages where ALL_SRC files reside.
+ALL_PKGS := $(sort $(shell go list ./...))
+
+GOTEST_OPT?= -race -timeout 120s
+GOCMD?= go
+GOTEST=$(GOCMD) test
+GO_ACC=go-acc
+LINT=golangci-lint
+IMPI=impi
+
+GOOS := $(shell $(GOCMD) env GOOS)
+GOARCH := $(shell $(GOCMD) env GOARCH)
+GH := $(shell which gh)
+
+.PHONY: test
+test:
+	$(GOTEST) $(GOTEST_OPT) ./...
+
+.PHONY: test-with-cover
+test-with-cover:
+	$(GO_ACC) --output=coverage.out ./...
+
+.PHONY: benchmark
+benchmark:
+	$(GOTEST) -bench=. -benchtime=5s -count 5 -benchmem -cpuprofile=cpu.out -memprofile=mem.out -run=notests ./
+
+.PHONY: fmt
+fmt:
+	gofmt -w -s ./
+	goimports -w  -local go.opentelemetry.io/collector ./
```

Comment:
```suggestion
	goimports -w  -local github.com/open-telemetry/opentelemetry-lambda/collector ./
```

Reply:
Thanks for catching, guess its easy to see where I copy/pasta'd from :D